services/auth-query-service/AuthQueryService.Api/.env.example

```
@@ -0,0 +1,65 @@
+RABBITMQ_USERNAME=guest
+RABBITMQ_PASSWORD=guest
+RABBITMQ_VIRTUALHOST=/
+
+# ================================================
+# JWT Configuration
+# ================================================
+# Must match Auth Service JWT configuration
+JWT_KEY=THIS_IS_A_MUCH_LONGER_AND_MORE_SECURE_SUPER_SECRET_KEY_123456789
+JWT_ISSUER=Soundmates.Auth
+JWT_AUDIENCE=Soundmates.Users
+JWT_EXPIRES_IN_HOURS=48
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_EXPIRES_IN_HOURS key should go before the JWT_ISSUER key

(UnorderedKey)